web/src/message_store.ts

```diff
@@ -194,30 +196,44 @@ export function get_pm_full_names(user_ids: number[]): string {
     return names.join(", ");
 }
 
-export function set_message_booleans(message: Message): void {
+export function set_message_booleans(message: RawMessage): MessageWithBooleans {
```

Comment:
I wonder if a different function name would make sense here, since I don't feel like "set" is what we're doing anymore, but I couldn't think of a good alternative.

Reply:
I think probably we want something like `convert_raw_message_to_message`